roles/netbootxyz/templates/menu/k3os.ipxe.j2

```diff
@@ -1,24 +1,76 @@
 #!ipxe
 
-# k3OS
+# k3OS by Rancher
 # https://k3os.io/
+# https://github.com/rancher/k3os#configuration-reference
 
 goto ${menu} ||
 
 :k3os
-set os {{ releases.k3os.name }}
-set k3os_mirror {{ releases.k3os.mirror }}
```

Comment:
You'll still need to set this mirror variable once as it's pulling it from the defaults above.

Reply:
Ok I've added those variables